### PR TITLE
Build VSIXs on multiple platforms to parallelize build

### PR DESCRIPTION
**Pull request by dibarbet**
Building vsixs is the longest part of CI - takes around 20 minutes.

This splits the vsix build into per platform builds.  This as the added benefit of allowing us to do custom per platform steps in the future (e.g. codesign on macos).

### CODE REVIEW

File: tasks/offlinePackagingTasks.ts [hunk 1]
```diff
@@ -33,8 +32,14 @@ import path = require('path');
...
+    vsceTarget: string;
+    rid: string;
+    platformInfo: PlatformInformation;
+}
+
 // Mapping of vsce vsix packaging target to the RID used to build the server executable
-export const platformSpecificPackages = [
+export const platformSpecificPackages: VSIXPlatformInfo[] = [
     { vsceTarget: 'win32-x64', rid: 'win-x64', platformInfo: new PlatformInformation('win32', 'x86_64') },
     { vsceTarget: 'win32-ia32', rid: 'win-x86', platformInfo: new PlatformInformation('win32', 'x86') },
```
Comment (JoeRobich) [inline tasks/offlinePackagingTasks.ts:44]: Wonder how much longer we should build a win32 package. See https://code.visualstudio.com/updates/v1_84#_windows-32bit-support-ends

Comment (dibarbet) [reply tasks/offlinePackagingTasks.ts:44]: Ah interesting - yeah let me see.

**Comment by dibarbet** [comment]
Time comparison in build vsixes

Old:
image

New:
image

File: tasks/offlinePackagingTasks.ts (hunk 1 above)
Comment (dibarbet) [reply tasks/offlinePackagingTasks.ts:44]: It's under discussion - for now will leave this PR as-is.

File: .vscode/launch.json [hunk 2]
```diff
@@ -129,6 +129,17 @@
             ],
             "preLaunchTask": "buildDev"
         },
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Debug gulp task",
```
Comment (dibarbet) [inline .vscode/launch.json:135]: just allows you to launch a gulp task under the debugger

**Review by arkalyanms — APPROVED** [review]

**Pull request merged**